loopout.go

```diff
@@ -900,29 +914,47 @@ func (s *loopOutSwap) sweep(ctx context.Context,
 		return s.htlc.GenSuccessWitness(sig, s.Preimage)
 	}
 
+	var (
+		remainingBlocks    = s.CltvExpiry - s.height
+		blocksToLastReveal = remainingBlocks - MinLoopOutPreimageRevealDelta
+		preimageRevealed   = s.state == loopdb.StatePreimageRevealed
```

Comment:
micro nit: comes down to personal taste but for variables that have a value assigned, I prefer the `:=` syntax. Would also fix the slightly too long line.